dev/Breadcrumb/BreadcrumbBar.xaml

```diff
@@ -18,17 +20,22 @@
         <Setter Property="UseSystemFocusVisuals" Value="{StaticResource UseSystemFocusVisuals}" />
         <Setter Property="FocusVisualMargin" Value="-3" />
         <Setter Property="IsTabStop" Value="True"/>
+        <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}" />
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="local:BreadcrumbBarItem">
-                    <Grid x:Name="PART_LayoutRoot" CornerRadius="{TemplateBinding CornerRadius}" >
+                    <Grid x:Name="PART_LayoutRoot"
+                          contract7Present:CornerRadius="{TemplateBinding CornerRadius}" >
```

Comment:
>contract7Present [](start = 26, length = 16)

If contract7NotPresent then we should set the corner radius on panels to {ThemeResource ControlCornerRadius}

Reply:
Fixed